lib/facebook_ads/ad_objects/server_side/custom_data.rb

```diff
@@ -83,54 +87,46 @@ class CustomData
       # @param [Integer] num_items
       # @param [String] status
       # @param [String] search_string
-      def initialize(value: nil,
-                     currency: nil,
-                     content_name: nil,
-                     content_category: nil,
-                     content_ids: nil,
-                     contents: nil,
-                     content_type: nil,
-                     order_id: nil,
-                     predicted_ltv: nil,
-                     num_items: nil,
-                     status: nil,
-                     search_string: nil)
+      def initialize(**data)
 
-        unless value.nil?
-          self.value = value
+        unless data[:value].nil?
+          self.value = data.delete(:value)
```

Comment:
we're changing the use pattern of `data` here, are we confident that there are no impacts if we delete the keys out of the data hash?

Reply:
Decide to switch back